lib/api/core/entrypoints/embedded/protocols/http.js

```diff
@@ -201,6 +201,7 @@ class HttpProtocol extends Protocol {
           return;
         }
 
+        request.response.headers['Content-Length'] = String(deflated.length);
```

Comment:
Here and for all other usages: shouldn't we rather use the response's `setHeader` method to handle the casing? (i.e. `content-Length`, `content-length`...)

Reply:
Forgot about that, good catch. I should also use getHeader when fetching a header (in unit tests).